### ISSUE TITLE
Wrong value for .Type in connection segment (and still no .SSID value)

### ISSUE THREAD

**Issue by JensGJ** [JanDeDobbeleer/oh-my-posh#3077]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Following up on #3049 that was marked as solved in v.12.13.2, I now have a slightly different problem with the connection segment. 

It seems to incorrectly identify the connection type. When I am on WIFI the connection type is still "ethernet" and there is no value present in the .SSID property. 

For debugging I made the following template: 

`"template": "Type: '{{ .Type }}' SSID: '{{ .SSID }}' Name: '{{ .Name }}'"`

Which currently yields: 

`Type: 'ethernet' SSID: '' Name: 'Intel(R) Ethernet Controller (3) I225-LM'`

Although no network cable is plugged in and I have connection through the wi-fi adapter: 

```
Get-NetAdapter | ft Name, InterfaceDescription, Status

Name                         InterfaceDescription                                                             Status
----                         --------------------                                                             ------
Ethernet 5                   Intel(R) Ethernet Controller (3) I225-LMvP                                       Disconnected
Ethernet 3                   Cisco AnyConnect Secure Mobility Client Virtual Miniport Adapter for Windows x64 Not Present
Ethernet                     Intel(R) Ethernet Controller (3) I225-LM                                         Disconnected
Wi-Fi                        Intel(R) Wi-Fi 6E AX210 160MHz                                                   Up
Bluetooth Network Connection Bluetooth Device (Personal Area Network)                                         Disconnected
```

On my secondary machine the behavior is the same. If I do connect by cable it shows the expected values (the same as shown earlier - so essentially the problem seems to be that the Type property is "ethernet" when I'm connected to the wi-fi adapter.

### Theme

```
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#003543",
          "foreground": "#00c983",
          "leading_diamond": "\ue0b6",
          "style": "diamond",
          "type": "os"
        },
        {
          "background": "#DA627D",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "folder_icon": "\uf115",
            "folder_separator_icon": "\\",
            "home_icon": "\uf7db",
            "mapped_locations": {
              "C:\\Dev\\Git": "\uf09b",
              "C:\\Users\\jgj.it\\OneDrive - CBS - Copenhagen Business School" : "\uf8c9",
              "C:\\Users\\jgj.it\\Dropbox" : "\uf6e2"

            },
            "style": "agnoster_full",
            "windows_registry_icon": "\ue0b1"
          },
          "style": "powerline",
          "type": "path"
        },
        {
          "background": "#FCA17D",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#FFEB3B{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#FFCC80{{ end }}",
            "{{ if gt .Ahead 0 }}#B388FF{{ end }}",
            "{{ if gt .Behind 0 }}#B388FB{{ end }}"
          ],
          "foreground": "#ffffff",
          "foreground_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#444{{ end }}"
          ],
          "powerline_symbol": "\ue0b0",
          "properties": {
            "branch_icon": " <#ffffff>\ue0a0 </>",
            "display_status": true,
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_upstream_icon": true
          },
          "style": "powerline",
          "type": "git"
        },
        {
          "background": "#043452",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "type": "poshgit"
        },
        {
          "background": "#76b367",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "type": "node"
        },
        {
          "background": "#83769c",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "always_enabled": true
          },
          "style": "powerline",
          "type": "executiontime"
        },
        {
          "background": "#33658A",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "time_format": "15:04"
          },
          "style": "powerline",
          "type": "time"
        },
        {
          "type": "connection",
          "style": "powerline",
          "background": "#22eead",
          "foreground": "#222222",
          "powerline_symbol": "\uE0B0",
          "template": "Type: '{{ .Type }}' SSID: '{{ .SSID }}' Name: '{{ .Name }}'"

        },
        {
          "type": "wifi",
          "style": "powerline",
          "background": "#8822ee",
          "foreground": "#222222",
          "background_templates": [
            "{{ if (lt .Signal 60) }}#DDDD11{{ else if (lt .Signal 90) }}#DD6611{{ else }}#11CC11{{ end }}"
          ],
          "powerline_symbol": "\uE0B0",
          "template": "\uFAA8 {{ .SSID }} {{ .Signal }}% {{ .ReceiveRate }}Mbps\u2800"
        },
        {
          "background": "#2e9599",
          "background_templates": [
            "{{ if gt .Code 0 }}red{{ end }}"
          ],
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "always_enabled": true
          },
          "style": "diamond",
          "trailing_diamond": "\ue0b4",
          "type": "exit"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "#cd5e42",
          "style": "plain",
          "type": "root"
        },
        {
          "foreground": "#CD4277",
          "style": "plain",
          "type": "text"
        }
      ],
      "type": "prompt"
    }
  ],
  "final_space": true,
  "version": 2
}

```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 12.13.2

Segments:

ConsoleTitle(false)  -   0 ms -
os(true)             -   0 ms -  
path(true)           -   0 ms -  
git(false)           -   6 ms -
poshgit(false)       -   0 ms -
node(false)          -   0 ms -
executiontime(true)  -   0 ms -  0ms
time(true)           -   1 ms -  09:33
connection(true)     -   3 ms - Type: 'ethernet' SSID: '' Name: 'Intel(R) Ethernet Controller (3) I225-LM'
wifi(false)          -   0 ms -
exit(true)           -   0 ms -   
root(false)          -   0 ms -
text(false)          -   0 ms -

Run duration: 26.2745ms

Cache path: C:\Users\jgj.it\AppData\Local\oh-my-posh

Config path: C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json

Logs:

2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\AppData\Local
2022/11/14 09:33:12 Getenv duration: 0s, args: LOCALAPPDATA
2022/11/14 09:33:12 CachePath duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json
2022/11/14 09:33:12 Getenv duration: 0s, args: POSH_THEME
2022/11/14 09:33:12 debug: Shell
process name: pwsh.exe
2022/11/14 09:33:12 Shell duration: 11.4226ms, args:
2022/11/14 09:33:12 resolveConfigPath duration: 11.4226ms, args:
2022/11/14 09:33:12 Init duration: 11.4226ms, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 config.loadConfig duration: 719.4µs, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv

2022/11/14 09:33:12 Getenv duration: 0s, args: OMP_CACHE_DISABLED
2022/11/14 09:33:12 WindowsRegistryKeyValue duration: 0s, args: HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: WindowsRegistryKeyValue
ColorizationColor(DWORD): 0xC44967AA
2022/11/14 09:33:12 Root duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 ErrorCode duration: 0s, args:
2022/11/14 09:33:12 IsWsl duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: User
jgj.it
2022/11/14 09:33:12 User duration: 0s, args:
2022/11/14 09:33:12 debug: Host
CBS12192
2022/11/14 09:33:12 Host duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 IsWsl duration: 0s, args:
2022/11/14 09:33:12 StackCount duration: 0s, args:
2022/11/14 09:33:12 DirIsWritable duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: CommandPath
C:\Program Files\Git\cmd\git.exe
2022/11/14 09:33:12 CommandPath duration: 6.3569ms, args: git.exe
2022/11/14 09:33:12 HasCommand duration: 6.3569ms, args: git.exe
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 error: HasParentFilePath
CreateFile C:\/.git: The system cannot find the file specified.
2022/11/14 09:33:12 HasParentFilePath duration: 0s, args: .git
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 ExecutionTime duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 wifiNetwork duration: 0s, args:
2022/11/14 09:33:12 error: network
Network type 'wifi' not found
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 ErrorCode duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 Root duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\AppData\Local
2022/11/14 09:33:12 Getenv duration: 0s, args: LOCALAPPDATA
2022/11/14 09:33:12 CachePath duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:Version: 12.13.2

Segments:

ConsoleTitle(false)  -   0 ms -
os(true)             -   0 ms -  
path(true)           -   0 ms -  
git(false)           -   6 ms -
poshgit(false)       -   0 ms -
node(false)          -   0 ms -
executiontime(true)  -   0 ms -  0ms
time(true)           -   1 ms -  09:33
connection(true)     -   3 ms - Type: 'ethernet' SSID: '' Name: 'Intel(R) Ethernet Controller (3) I225-LM'
wifi(false)          -   0 ms -
exit(true)           -   0 ms -   
root(false)          -   0 ms -
text(false)          -   0 ms -

Run duration: 26.2745ms

Cache path: C:\Users\jgj.it\AppData\Local\oh-my-posh

Config path: C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json

Logs:

2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\AppData\Local
2022/11/14 09:33:12 Getenv duration: 0s, args: LOCALAPPDATA
2022/11/14 09:33:12 CachePath duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json
2022/11/14 09:33:12 Getenv duration: 0s, args: POSH_THEME
2022/11/14 09:33:12 debug: Shell
process name: pwsh.exe
2022/11/14 09:33:12 Shell duration: 11.4226ms, args:
2022/11/14 09:33:12 resolveConfigPath duration: 11.4226ms, args:
2022/11/14 09:33:12 Init duration: 11.4226ms, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 config.loadConfig duration: 719.4µs, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv

2022/11/14 09:33:12 Getenv duration: 0s, args: OMP_CACHE_DISABLED
2022/11/14 09:33:12 WindowsRegistryKeyValue duration: 0s, args: HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: WindowsRegistryKeyValue
ColorizationColor(DWORD): 0xC44967AA
2022/11/14 09:33:12 Root duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 ErrorCode duration: 0s, args:
2022/11/14 09:33:12 IsWsl duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: User
jgj.it
2022/11/14 09:33:12 User duration: 0s, args:
2022/11/14 09:33:12 debug: Host
CBS12192
2022/11/14 09:33:12 Host duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 PathSeparator duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 IsWsl duration: 0s, args:
2022/11/14 09:33:12 StackCount duration: 0s, args:
2022/11/14 09:33:12 DirIsWritable duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: CommandPath
C:\Program Files\Git\cmd\git.exe
2022/11/14 09:33:12 CommandPath duration: 6.3569ms, args: git.exe
2022/11/14 09:33:12 HasCommand duration: 6.3569ms, args: git.exe
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 error: HasParentFilePath
CreateFile C:\/.git: The system cannot find the file specified.
2022/11/14 09:33:12 HasParentFilePath duration: 0s, args: .git
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 ExecutionTime duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 wifiNetwork duration: 0s, args:
2022/11/14 09:33:12 error: network
Network type 'wifi' not found
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 ErrorCode duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 Root duration: 0s, args:
2022/11/14 09:33:12 debug: Pwd
C:\Users\jgj.it
2022/11/14 09:33:12 Pwd duration: 0s, args:
2022/11/14 09:33:12 debug: Home
C:\Users\jgj.it
2022/11/14 09:33:12 GOOS duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 TemplateCache duration: 0s, args:
2022/11/14 09:33:12 Shell duration: 0s, args:
2022/11/14 09:33:12 debug: Getenv
C:\Users\jgj.it\AppData\Local
2022/11/14 09:33:12 Getenv duration: 0s, args: LOCALAPPDATA
2022/11/14 09:33:12 CachePath duration: 0s, args:
2022/11/14 09:33:12 Flags duration: 0s, args:
```

**Comment by JanDeDobbeleer** [comment]
@JensGJ can you try again on the latest version?

**Comment by JensGJ** [comment]
Sure. There is no new version available on Chocolatey - but I've made the fix change shown in commit 6ab687553dad5128a0cc7fe6a7fba265b44c2435 in my local copy - and that works very nice:

![image](https://user-images.githubusercontent.com/42742575/202180830-a7f83eef-520b-4ee6-9e75-98062b5ff336.png)

**Comment by JanDeDobbeleer** [comment]
@JensGJ can you close if you're happy with the result?

**Comment by JensGJ** [comment]
Yes. Will the change be available in the next common release? (12.13.4?)

**Comment by JanDeDobbeleer** [comment]
Yes, it already is. You can even download the installer in releases here. The chocolatey installer isn't maintained by us.
